### ISSUE TITLE
East Asian Paragraph line-break issue (cause identified in XML)

### ISSUE THREAD

**Issue by henry132109** [dolanmiu/docx#713]
In runs of East Asian texts, usually no space is added between characters, but when a space is added, the remaining chunk is treated as 1 word as in European languages I guess and this happens:
<img width="505" alt="bug" src="https://user-images.githubusercontent.com/26480299/101788857-8d9e4080-3b3b-11eb-814b-c762c8b3e534.png">
Expected output:
![image](https://user-images.githubusercontent.com/26480299/101790050-f803b080-3b3c-11eb-9a7b-0c2415045340.png)

The code involved:
```
const dx = require('docx');

const test = async () => {
  try{
    const doc = new dx.Document({
      creator: process.env.SCHOOL_NAME,
      title: "TEST",
      description: "",
      styles: {
        paragraphStyles: [
          {
            id: "normal",
            name: "Normal",
            basedOn: "Normal",
            next: "Normal",
            run: {
              size: 24,
            },
          },
        ]
      }
    });

    //封面

    doc.addSection({
      children: [
        new dx.Paragraph({
          text: "1. 時取香有好開小量利推先定小我會。學的要了去意不備第魚！元才界物們；結動兩乎半合家了樂生：靜開需有聲是不大上所、落大於表球酒曾該媽決主完一後然輪出是。頭了新我媽然遠……了式員：車任原，足位許是走所足家環明子我電德。系交國斷而對約十一微毛走答出灣我爭，生裝重三層動達快緊天不次全山少和事演立然農水金需，比流是外進車樣是形部積少把我其，然身出禮縣一面市入工以再指集人面此走條目，話送的策常草大簡合上大字。",
        }),
        new dx.Paragraph({
          text: "1. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam id gravida diam. Donec faucibus quis ex quis tempus. Aliquam posuere ligula quis bibendum pellentesque. Vestibulum commodo nec nibh at maximus. Proin egestas elit ligula, eu lobortis sem cursus id. Quisque congue faucibus massa at posuere. Donec eu diam nisi. Suspendisse blandit nisl pulvinar lorem mattis tempor. Nam ac felis in magna molestie placerat at sit amet lacus. Fusce imperdiet ante arcu. Ut efficitur ante urna, ut rutrum magna consectetur viverra. Nunc a imperdiet mi. Nam vitae enim eget nisi commodo euismod eu egestas tellus. Donec nec vehicula sapien, vel dictum nisi.",
        }),
      ],
    });

    const b64string = await dx.Packer.toBase64String(doc);
    return b64string;
  }catch(err){
    console.log(err);
  }
}
```
### Possible solution:
Upon comparing the XML generated via this package and what I get from Microsoft Word, I discovered that changing `<w:rPrDefault/>` in word/styles.xml to `<w:rPrDefault><w:rFonts w:asciiTheme="minorHAnsi" w:eastAsiaTheme="minorEastAsia" w:hAnsiTheme="minorHAnsi" w:cstheme="minorBidi"/></w:rPrDefault>` successfully fixed the line break issue.

**Comment by henry132109** [comment]
I've checked: this issue still exists in 5.4.1.

**Comment by ehdwns980416** [comment]
+1
Thx for open this issue. 👍

**Comment by dolanmiu** [comment]
Will be fixed with: https://github.com/dolanmiu/docx/pull/731

**Comment by dolanmiu** [comment]
Created a demo which adds:

```xml
<w:rPrDefault><w:rFonts w:asciiTheme="minorHAnsi" w:eastAsiaTheme="minorEastAsia" w:hAnsiTheme="minorHAnsi" w:cstheme="minorBidi"/></w:rPrDefault>
```

https://github.com/dolanmiu/docx/blob/master/demo/53-chinese.ts

This feature is available currently in 5.5.0
